beacon-chain/rpc/service.go

```diff
@@ -132,6 +133,7 @@ type Config struct {
 	BlockNotifier           blockfeed.Notifier
 	OperationNotifier       opfeed.Notifier
 	StateGen                *stategen.State
+	StateSummaryCache       *cache.StateSummaryCache
```

Comment:
Can you look through all these stateSummaryCache additions and make sure they're all needed? Looks like theres only one usage of it.

Reply:
i looked. they are all required